hooks/install.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+HOOKSDIR=./.git/hooks/
+
+cp ./hooks/pre-commit "$HOOKSDIR/pre-commit"
```

Comment:
Not better of add that newline at the end?

Reply:
having a newline at the end of a file is of practically no importance

Reply:
Is looks ugly, enough reason IMHO

Reply:
Back in the old days, I used to have some problems with the missing last newline ;-)

Reply:
>It looks ugly

lol, only in the way Github displays it ;)